### DISCUSSION TITLE
Battery icon on taskbar disappears after hibrenation

### DISCUSSION THREAD

**Discussion by T0RNATO** [valinet/ExplorerPatcher#1581] [General]
Windows 11, OS Build 10.0.22621.1848.

Restarting explorer through the properties box makes it reappear.

**Comment by pyrates999** [comment]
Does it do this every time?

**Reply by T0RNATO** [reply]
Hmm... I thought it did, but after manually hibrenating and turning it on again it didn't - it appears to only do it when my laptop automatically does it while asleep to save power.

**Reply by pyrates999** [reply]
Ok, might be a bug in windows then.
